tools/nl/embeddings/README.md

```diff
@@ -52,6 +52,16 @@ This is a common sheet across the different index sizes.
    ```bash
    /run.sh -l medium <path_to_local_model>
    ```
+
+   To create custom embeddings (using the finetuned model in PROD):
+   ```bash
+   ./run.sh -c <embeddings_size> <local_sheets_csv_filepath> <worksheet_name>
```

Comment:
What sheets url does this worksheet belong to? If you're recommending providing the worksheet name like this, then you must also ask for the sheets url, no (instead of using the local downloaded path?)

Reply:
Ok, I am still a bit unsure what this means. I'll approve to unblock you though ;) Could you please document in the README exactly what supplying the local csv path means and what it means if you don't provide it etc? Thanks!

Reply:
sorry actually thinking about it more, I think it doesn't matter to supply the local path, so I'll remove it!